### PR TITLE
Batch Aggregation mailer

### PR DESCRIPTION
**Pull request by zwolf**
If the params to AggregationsController#update include a change to status, send an email to the associated user that includes the links to the output files if the run was successful. "Complete" here means either succeeded or failed.

# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

File: app/workers/aggregation_completed_mailer_worker.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class AggregationCompletedMailerWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :data_high
+
+  def perform(agg_id)
+    aggregation = Aggregation.find(agg_id)
+    AggregationCompletedMailer.aggregation_complete(aggregation).deliver
```
Comment (yuenmichelle1) [inline app/workers/aggregation_completed_mailer_worker.rb:10]: I think `.deliver` might be a deprecated method replaced by `deliver_now` or `deliver_later`

**Review by yuenmichelle1 — APPROVED** [review]
Changes look fine. I have one minor comment.

File: app/workers/aggregation_completed_mailer_worker.rb (hunk 1 above)
Comment (zwolf) [reply app/workers/aggregation_completed_mailer_worker.rb]: apparently so! Every other mailer uses it, but I'll just change it here and leave that refactor for later.
